app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -572,6 +576,93 @@ export default class SampleViewV2 extends React.Component {
     return {};
   };
 
+  countReportRows = () => {
+    const { filteredReportData, reportData } = this.state;
+    let total = reportData.length;
+    let filtered = filteredReportData.length;
+    reportData.forEach(genusRow => {
+      total += genusRow.species.length;
+      filtered += genusRow.filteredSpecies.length;
+    });
+    return { total, filtered };
+  };
+
+  filteredMessage = () => {
+    const { total, filtered } = this.countReportRows();
+    return (
+      filtered != total &&
```

Comment:
The current version of the report displays the filtered rows message even if no filters are selected. Do we want to carry this over so a user can immediately see how many rows are in the entire table?
![image](https://user-images.githubusercontent.com/53838890/69571672-7237b600-0f77-11ea-94c7-c4bee739f302.png)

Reply:
Perhaps we can just say `${total} total rows` if there are no filters and leave off the filter-related text.

Reply:
Done